packages/zosfiles/src/methods/download/doc/IDownloadOptions.ts

```diff
@@ -103,4 +108,12 @@ export interface IDownloadOptions extends Omit<IDownloadSingleOptions, "stream">
      * Specifies whether hidden files whose names begin with a dot should be downloaded.
      */
     includeHidden?: boolean;
+
+    /**
+     * An optional pattern for restricting the response list
+     */
+    pattern?: string;
+
+    memberPatternResponse?: any;
```

Comment:
Can we had a short description for this new optional property in the interface? 😋